Zend/zend_closures.c

```diff
@@ -428,6 +434,11 @@ ZEND_API const zend_function *zend_get_closure_method_def(zend_object *obj) /* {
 ZEND_API zval* zend_get_closure_this_ptr(zval *obj) /* {{{ */
 {
 	zend_closure *closure = (zend_closure *)Z_OBJ_P(obj);
+	
+	if (closure->func.common.fn_flags & ZEND_ACC_ABSTRACT) {
+	    return NULL;
```

Comment:
Hm, shouldn't this be returning the $this for partials as well? Again, similar to fake closures?